app/models/generic_object.rb

```diff
@@ -17,6 +17,9 @@ class GenericObject < ApplicationRecord
            :custom_actions, :custom_action_buttons,
            :to => :generic_object_definition, :allow_nil => true
 
+  delegate :name, :to => :generic_object_definition,       :prefix => true, :allow_nil => false
+  virtual_column  :generic_object_definition_name,         :type => :string
```

Comment:
Is a column required by report?  Seems it can be displayed well even without this line.

Reply:
I believe this line is for the API to return the class name for use by the Service UI.

Reply:
The API can leverage this, yes.
But we do need this virtual column specified in the `GenericObject.yaml` that the Generic Object Listview uses to display the columns - https://github.com/ManageIQ/manageiq-ui-classic/blob/master/product/views/GenericObject.yaml#L23